core/modules/layout/layout.api.php

```diff
@@ -155,16 +151,14 @@ function hook_layout_style_info() {
   $info['custom_style'] = array(
     'title' => t('A new style'),
     'description' => t('An advanced style with settings.'),
-    // The theme key indicating what theme function/template will be used.
-    'region theme' => 'mymodule_layout_region',
     // The theme key for rendering an individual block.
     'block theme' => 'mymodule_block',
     // Provide a class name if this style has settings. The class should extend
     // the LayoutStyle class.
     'class' => 'MyModuleLayoutStyle',
     // If nesting templates or a theme include file in a directory, the relative
     // path can be specified.
-    'path' => 'templates',
+    'template' => 'templates/mymodule-file.tpl.php',
```

Comment:
Revert this, `path` is still valid. Add a separate entry for `template` if desired.